int32 width
int32 height
int32 framerate_numerator
int32 framerate_denominator
float32 exposure
